backend/python/app/models/__init__.py

```diff
@@ -3,6 +3,9 @@
 db = SQLAlchemy()
 erase_db_and_sync = False
 
+from .user import User
+from .entity import Entity
+
```

Comment:
Can either one of the sets for the User/Entity imports be removed?

Reply:
It kiiinda helps with imports. With these two lines, this is allowed:
```
from ...models import Entity as EntityModel
```
Without these lines you have to do:
```
from ...models.entity import Entity as EntityModel
```
honestly, now that I look at it more, I think the latter is more clear. I'm removing these lines now.